### DISCUSSION TITLE
How about if I already have an existing DB and I want to use it?

### DISCUSSION THREAD

**Discussion by JenuelDev** [tekartik/sqflite#1148] [Q&A]
How to handle already existing Sqlite db file. So I have a dictionary DB and I wanted to use that existing DB how do I handle this? THanks

**Comment by alextekartik** [comment]
It should work. One solution is to put your db in an asset then upon start, copy the asset as a regular file and open it.
More information here: https://github.com/tekartik/sqflite/blob/master/sqflite/doc/opening_asset_db.md

**Reply by JenuelDev** [reply]
Thanks for your answer mate. Appreciate it. 👍
